README.md

```diff
@@ -256,12 +259,31 @@ var helper = client.paginate(
 )
 ```
 
+##### queryTimeout
 ```javascript
 var data = client.query(q.Paginate(q.Collections()), {
   queryTimeout: 100,
 })
 ```
 
+##### traceparent
+A [W3C-compliant](https://w3c.github.io/trace-context) identifier for enabling distributed tracing across different
+vendors. If not provided, one is automatically generated server-side and attached to the query. See
```

Comment:
Let customers know if they submit an invalid traceparent, they must override their provided one with the one returned in our response, since theirs would be silently dropped.

Reply:
Updated